lib/models/emailAction.js

```diff
@@ -52,8 +52,13 @@ function SendMail(action, event, callback) {
 
         transporter.sendMail(mailOptions, function(err, info) {
             logger.debug('emailAction.SendMail %j %j %j', mailOptions, err, info);
+            // Not a HTTP request, must count outgoingTransaction
```

Comment:
Mmmm... not fully

It seems 2/3 occurence were changed, but not the one on which this comment has been created...